src/test/java/org/example/c20debugging/BankAccountTestParameterized.java

```diff
@@ -0,0 +1,49 @@
+package org.example.c20debugging;
+
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+
+import java.util.Arrays;
+import java.util.Collection;
+
+import static junit.framework.TestCase.assertEquals;
+
+@RunWith(Parameterized.class)
```

Comment:
Use Junit5 version of parametrized tests. See https://www.baeldung.com/parameterized-tests-junit-5

Reply:
done